Add unittest section to the documentation (#19276)
DataDog/documentation: content/en/continuous_integration/tests/setup/python.md

    assert result == 25
```

### Using unittest

To enable instrumentation of `unittest` tests, run your tests by appending `ddtrace-run` to the beginning of your `unittest` command.

Make sure to specify the name of the service or library under test in the `DD_SERVICE` environment variable.
Additionally, you may declare the environment where tests are being run in the `DD_ENV` environment variable:

{{< code-block lang="shell" >}}
DD_SERVICE=my-python-app DD_ENV=ci ddtrace-run python -m unittest
{{< /code-block >}}

Alternatively, if you wish to enable `unittest` instrumentation manually, use `patch()` to enable the integration:

{{< code-block lang="python" >}}
from ddtrace import patch
import unittest
patch(unittest=True)

class MyTest(unittest.TestCase):
    def test_will_pass(self):
        assert True
{{< /code-block >}}

#### Known limitations

> In some cases, if your `unittest` test execution is being ran in a parallel manner, this may break the instrumentation and affect test visibility.

Reviewer: ```suggestion
In some cases, if your `unittest` test execution is run in a parallel manner, this may break the instrumentation and affect test visibility.
```

> It is recommended to use up to one process at a time to prevent affecting test visibility.

Reviewer: ```suggestion
Datadog recommends you use up to one process at a time to prevent affecting test visibility.
```
